junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java

```diff
@@ -136,7 +149,20 @@ public void postProcessTestInstance(Object testInstance, ExtensionContext contex
 
         // at this point we can be sure that either no or exactly one WeldInitiator was found
         if (initiator == null) {
-            initiator = WeldInitiator.from(AbstractWeldInitiator.createWeld().addPackage(false, testInstance.getClass())).build();
+            Weld weld = createWeld();
+            Class<?> testClass = testInstance.getClass();
+            @SuppressWarnings("unchecked")
+            List<WeldCustomizer> customizers = (List<WeldCustomizer>) context.getRoot().getStore(Namespace.GLOBAL).get(WELD_CUSTOMIZERS);
```

Comment:
Why global store? We should use our own namespace.
I understand you wanted something which is not test-instance specific, so maybe we should tweak/add helper methods we have to allow for custom namespace?

Reply:
`Namespace.create(WeldJunit5Extension.class)` would be ok?

Reply:
Now that I think about it - you are always storing this information again in `beforeAll`. If you store this in a "global" manner, you should be able to load it once a just retrieve the list for every test instance executed, right?

Reply:
Hm, I thought that `BeforeAllCallback` is something like "before suite"... So is the store returned from `context.getRoot().getStore(NAMESPACE)` global, i.e. shared accross all test instances? Javadoc is not clear.

Reply:
`BeforeAll` is before all tests in that instance (as opposed to `BeforeEach`). Extensions cannot hook before whole suite from what I know. As for the store, I am gonna double check the docs myself...

Reply:
It does not seem to be the case. I get different `Store` object for every test class. That's odd.